R/globals.R

```diff
@@ -9,12 +9,11 @@ out.dir <- data.dir                     # may want to write to a separate dir to
 
 # Input files
 gwas.fn <- lapply(c(bed='bed',bim='bim',fam='fam',gds='gds'), function(n) sprintf("%s/GWAStutorial.%s", data.dir, n))
-clinical.fn <- sprintf("%s/GWAStutorial_clinical.csv", data.dir)
-onethou.fn = lapply(c(info='info',ped='ped'), function(n) sprintf("%s/chr16_1000g_CEU.%s", data.dir, n))
-protein.coding.coords.fname <- sprintf("%s/ProCodgene_coords.csv", data.dir)
+clinical.fn <- sprintf("%s/GWAStutorial_clinical.csv", data.dir) 
+onethou.fn <- lapply(c(info='info',ped='ped'), function(n) sprintf("%s/chr16_1000g_CEU.%s", data.dir, n))
+protein.coding.coords.fname <- sprintf("%s/ProCodgene_coords.csv", out.dir)
```

Comment:
I'm confused.  You changed this back to out.dir after I set it to data.dir, I think.
I thought your previous PR pointed out that protein.coding.coords.fname was an *input* and therefore located in `data.dir`.  `out.dir` is supposed to be a location possibly different than `data.dir` where results are written.